torch/jit/_monkeytype_config.py

```diff
@@ -124,3 +127,29 @@ def __init__(self):
             pass
 
     monkeytype_trace = None  # noqa: F811
+
+def jit_code_filter(code: CodeType) -> bool:
+    """
+    Custom CodeFilter for Torchscript to trace forward calls
+    The code filter is similar to default code filter for monkeytype and
+    excludes tracing of stdlib and site-packages.
+    """
+    # Filter code without a source file
+    if code.co_name != 'forward' and (not code.co_filename or code.co_filename[0] == '<'):
```

Comment:
Can you call `default_code_filter` inside `jit_code_filter` to avoid code duplication?

Reply:
Can we devise a test covering this custom code filter?

Reply:
The available tests should be sufficient enough to validate the code filter since we are now by-passing the check `not code.co_filename or code.co_filename[0] == '<'`  for all forward calls.

Reply:
> Can you call `default_code_filter` inside `jit_code_filter` to avoid code duplication?

I agree there is a bit of duplication. Since this check needs to happen together `if code.co_name != 'forward' and (not code.co_filename or code.co_filename[0] == '<'):` I am not sure how I can avoid this duplication.

Reply:
I see. In that case, maybe we don't really need the logic that's related to checking `MONKEYTYPE_TRACE_MODULES` because it is specific to `default_code_filter` (note that else branch on line 161 is still needed to skip non-user code libraries.

Reply:
Agreed. Done